lib/gtfs/route_pattern.ex

```diff
@@ -37,4 +38,8 @@ defmodule Gtfs.RoutePattern do
       representative_trip_id: row["representative_trip_id"]
     }
   end
+
+  @spec in_route_id_set?(Csv.row(), MapSet.t(Route.id())) :: boolean
+  def in_route_id_set?(row, route_id_set),
```

Comment:
Do you think having `row` or `csv` somewhere in this function name would reduce risk of misusing it?
(Same for `Trip.in_route_id_set?`, `Timepoint.in_trip_id_set?`, and `Timepoint.includes_a_checkpoint?`)

Reply:
As in something like, `csv_row_in_route_id_set?/2`?

Reply:
Updated

Reply:
I'd lean toward the shorter `row_in_route_id_set?` or `row_has_timepoint?`, but yeah.